File: src/main/java/nz/laspruca/tcplugin/commands/Bee.java
package nz.laspruca.tcplugin.commands;

import org.bukkit.command.*;
import org.jetbrains.annotations.*;

@nz.laspruca.tcplugin.Command(name = "bee")
public class Bee implements CommandExecutor {
	@Override
	public boolean onCommand(CommandSender commandSender, @NotNull Command command, @NotNull String s, String[] strings) {
		commandSender.sendMessage("According to all known laws of aviation, there is no way that a bee should be able to fly.");
		return true;
	}
}

File: src/main/kotlin/nz/laspruca/tcplugin/commands/Bee.kt
package nz.laspruca.tcplugin.commands

import org.bukkit.command.Command
import org.bukkit.command.CommandSender

import org.bukkit.command.CommandExecutor


class Bee : CommandExecutor {
    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<out String>): Boolean {
        sender.sendMessage("According to all known laws of aviation, there is no way that a bee should be able to fly.")
        return true
    }
}
